src/config/config.ts

```diff
@@ -69,4 +71,14 @@ export interface Config {
    * Sandbox feature configuration
    */
   sandboxFeatures?: SandboxFeatures;
+
+  /**
+   * Name of a company that adopted Nplan.
+   * This is used e.g. for:
+   *    CustomStyle, when determining the relevant custom style class;
+   *    CustomLogo;
+   *    favicon;
+   *    overriding translations (case appTitle)
+   */
+  partnerCompany?: string;
```

Comment:
Wasn't sure how to name this, let me know if there is a more fitting term

Reply:
Yup, for something company-specific like styles or some translation overrides. Currently looks like: `/ext/Fintraffic/<feature or asset name>`

Reply:
So maybe `whitelabelAssetPath`  or something similar? It highlights what it is used for better I think.

Reply:
There was one more kind of use case for partnerCompany, for example in: 
```
 useEffect(() => {
    getMessages(selectedLocale).then(async (messages) => {
      setMessages(() => messages);
      if (partnerCompany) {
        const extMessages = await getExtMessages(partnerCompany, selectedLocale)
        const combinedMessages = {
          ...messages,
          ...extMessages,
        };
        setMessages(() => combinedMessages);
      }
    });
  }, [selectedLocale, partnerCompany]);
```
So it appears also as some indicator that something is meant for non-Entur scenario only.

But this particular case can be reworked into having a SandboxFeature-based custom IntlProvider, while original EnkiIntlProvider then stays entirely intact and partnerCompany then gets renamed into something path related...